### PR TITLE
#102 add monospace support for textarea

### PR DESCRIPTION
**Pull request by elekktrisch**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [ x ] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] It's submitted to the `main` branch for v2.x (or to a previous version branch)
- [ x ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ x ] All tests are passing
- [ ] New/updated tests are included

If adding a **new feature**, the PR's description includes:
- [ x ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: libraries/ui-library/src/components/six-textarea/six-textarea.tsx [hunk 1]
```diff
@@ -109,6 +109,9 @@ export class SixTextarea {
   /** The textarea's inputmode attribute. */
   @Prop() inputmode?: 'none' | 'text' | 'decimal' | 'numeric' | 'tel' | 'search' | 'email' | 'url';
 
+  /** The textarea's inputmode attribute. */
```
Comment (elekktrisch) [inline libraries/ui-library/src/components/six-textarea/six-textarea.tsx:112]: Beschreibung korrigieren

**Review by colinscz — COMMENTED** [review]
- [ ] Resolve issue with git diff

**Comment by elekktrisch** [comment]
git diff resolved

**Comment by ibirrer** [comment]
Thanks for submitting a PR! One concern I have is the potential need to add this property to other components, like input. This might clutter the API. Have you considered using just CSS in your application? For instance, the following would achieve the same effect without adding an additional property:

```css 
six-textarea::part(textarea) {
  font-family: monospace;
}
```
Additionally, I'm not certain if our internal style guide has specified a monospace font yet. Could be that [Noto Sans Mono](https://fonts.google.com/noto/specimen/Noto+Sans+Mono) would be the better option?

**Comment by elekktrisch** [comment]
@ibirrer yes I did consider using CSS, but didn't know the ::part selector. That's most definitely the better option then. Thanks for the help!

**Pull request closed without merging**